### DISCUSSION TITLE
Quarto Diagrams - Mermaid documentation

### DISCUSSION THREAD

**Discussion by rmcd-mscb** [quarto-dev/quarto-cli#5219] [Feature Requests]
This is a small issue.  

As a new quarto user, the following text in the [diagrams documentation](https://quarto.org/docs/authoring/diagrams.html)  eluded me in that I kept using the `//|` rather than `%%/` in a mermaid block to define a figure caption.

`Note that the cell options syntax is slightly different for {mermaid} and {dot} (%%| and //|, respectively) because they have distinct character sequences for single-line comments.`

Suggest adding it as a call out for more emphasis and perhaps adding an example of adding a figure caption to a mermaid block.

**Comment by cscheid** [comment]
For what's worth, we do have mermaid metadata block examples in the [Sizing](https://quarto.org/docs/authoring/diagrams.html#sizing) and [Echo](https://quarto.org/docs/computations/execution-options.html) sections.

**Reply by rmcd-mscb** [reply]
Yup - thanks Carlos.  It's all there :)  Still I think adding that text describing the difference in options syntax in a call out, you already have "Note" in the text, may help some other newbies :)

**Reply by cscheid** [reply]
We are intentionally conservative about adding callouts, because they compete with one another for visual attention.

With that said, I did change the page to describe the comment syntax for mermaid and graphviz diagrams, and that should be updated shortly.

**Reply by rmcd-mscb** [reply]
Thank you Carlos.
